app/src/main/res/layout/activity_chat.xml

```diff
@@ -1,66 +1,144 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/wallpaper"
     tools:context=".Activity.ChatActivity">
 
+    <androidx.appcompat.widget.Toolbar
```

Comment:
You can also do this without adding toolbar manually  just giving an icon to the default toolbar

Reply:
just a comment

Reply:
I tried but that's not possible with the original Actionbar